docs/getting-started-guides/docker.md

```diff
@@ -55,6 +55,25 @@ Here's a diagram of what the final result will look like:
 ### Prerequisites
 
 1. You need to have docker installed on one machine.
+2. Your kernel should support memory and swap accounting. Ensure that the
```

Comment:
It's sort of an issue in both?  Docker probably shouldn't be returning a 500 from the remote API in this case, since the docker CLI ignores the error, but the kubelet should also try to workaround this by detecting ahead of time and not passing values (https://github.com/GoogleCloudPlatform/kubernetes/issues/9837#issuecomment-112292120)

I think it's fine to say "kubernetes requires" for now, since it def. is true.